runtime/domain.c

```diff
@@ -1278,6 +1303,10 @@ CAMLprim value caml_domain_spawn(value callback, value term_sync)
   pthread_t th;
   int err;
 
+  if (atomic_load_relaxed(&domains_exiting) != 0) {
```

Comment:
```suggestion
  if (atomic_load_relaxed(&domains_exiting)) {
```
is more readable? (feel free to ignore).